src/Symfony/Component/Console/Command/Command.php

```diff
@@ -605,6 +605,10 @@ public function getUsages()
      */
     public function getHelper($name)
     {
+        if (null === $this->helperSet) {
+            throw new LogicException(sprintf('Unable to retrieve helper "%s" because there is no HelperSet defined for this command. Did you forget to call %s::setHelperSet()?', $name, get_class($this)));
```

Comment:
The first part (setting the application) can as well be related to the command not being registered at the application (which will then set itself on the command).

Maybe something like:

> Did you forget to add your command to the application or to set the application on the command using the setApplication() method? You can also set the HelperSet directly using the setHelperSet() method.